apps/element-storybook/src/form-migration.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="BS4 Migration/Form Migration" />
+
+# Form Migration
+
+For the `@availity/element` components, we have chosen to move away from
+`formik` in favor of `react-hook-form`. For exiting forms utilizing
+`@availity/react` and `formik`, some refactor is required.
+
+## Why `react-hook-form`?
+
+After a successful internal implementation of an extremely long and
+complicated form using `react-hook-form`, we decided to move forward with
+it. `react-hook-form` is very lightweight and flexible. It also boasts a
+large community and is actively and regularly supported.
+
+## Installation
+
+```sh
+npm install react-hook-form
+```
+
+OR
+
+```sh
+yarn add react-hook-form
+```
+
+## Usage
+
+At the moment, @availity/element does not export a `<Form />` component.
+Instead, use the native `<form>` element. You can then use `react-hook-form`'s
+`useForm` hook to manage your form.
+
+You can find some examples [here](/docs/layouts-form--docs).
```

Comment:
idk if we want to go this far rn or save it for later, but for a alot of the migration docs I think it's useful to have code examples to compare the old with the new, in this case an availity-react form/component with an element form/component.

Reply:
Yeah, I was trying to avoid maintaining examples in two spots and documenting react-hook-form (since we don't own it and are using it as intended). However, it's probably a better user experience to include actual examples. I could use the feedback form as an example.